Teradata/covalent
# fix(): add @angular/platform-browser as a peerDep to highlight and markdown packages

## Description
Adds `@angular/platform-browser` as a peerDependency to both the highlight and markdown packages. Address issue of having to add `@npm//@angular/platform-browser` when using covalent highlight and markdown when building with bazel.

Related issue #1382 

#### Test Steps
* Run `npm run build:lib` to ensure library is built and version placeholders replaced

#### General Tests for Every PR


// cc @JulieMarie
